zerver/openapi/zulip.yaml

```diff
@@ -21372,6 +21389,12 @@ components:
             Is `null` for channels with no message history.
 
             **Changes**: New in Zulip 2.1.0.
+        is_recently_active:
+          type: boolean
+          description: |
+            Whether the channel has recent message activity. Clients should use this to implement [sorting inactive channels to the bottom](/help/manage-inactive-channels).
+
```

Comment:
This text should be wrapped to ~100 characters.

Reply:
Done.